README.md

```diff
@@ -28,6 +30,103 @@ git clone git@github.com:kamp-us/monorepo.git
 cd monorepo
 ```
 
+- Install dependencies.
+
+```sh
+npm install
+```
+
+### 2) Modifying `hosts` file
+
+- Add the following text block to [your `/etc/hosts` file](https://www.howtogeek.com/27350/beginner-geek-how-to-edit-your-hosts-file/).
+
+```text
+127.0.0.1 localhost.kamp.us
+127.0.0.1 pasaport.localhost.kamp.us
+127.0.0.1 pano.localhost.kamp.us
+127.0.0.1 sozluk.localhost.kamp.us
+127.0.0.1 gql.localhost.kamp.us
+```
+
+### 3) Setting up `.env` files
+
+- Duplicate `.env.example` files and rename them as `.env` in the following folders: `db/prisma`, `apps/gql`, `apps/kampus`, `apps/pasaport`.
+
+### 4) Prisma Setup and Database Configuration
+
+- Build required packages(`sozluk` and `prisma client`). (It may fail, but it's fine, continue with the next step.)
+
+```sh
+npx turbo build
+```
+
+- Move to `db/prisma` folder.
+
+```sh
+cd db/prisma
+```
+
+- Build/start the container. (You should have docker installed.)
+
+```sh
+docker-compose up --build
+```
+
+- Open up another terminal. Move to `db/prisma` folder. (docker-compose will keep running on previous terminal.)
+
+```sh
+cd db/prisma
+```
+
+- Generate artifacts. (e.g. Prisma Client)
+
+```sh
+npm run prisma:generate
+```
+
+- Push the Prisma schema state to the database.
+
+```sh
+npm run prisma:push
+```
+
+### 5) Running dev servers
+
+- Open up 3 terminals, and run the commands below on each.
+
+```sh
+npm run dev -w @kampus-apps/kampus
+```
+
+```sh
+npm run dev -w @kampus-apps/gql
+```
+
+```sh
+npm run dev -w @kampus-apps/pasaport
+```
+
+- Now you can go to [localhost.kamp.us:3000](localhost.kamp.us:3000) to see it live. 🚀
+
+### 6) Logging in with GitHub on development server(optional)
+
+- Go to [OAuth Apps page](https://github.com/settings/developers) on GitHub.
```

Comment:
if we will put GitHub let's add all the applications here. 
- Twitch is missing
- Discord is missing